src/Microsoft.AspNetCore.Cors/Infrastructure/CorsPolicy.cs

```diff
@@ -141,5 +154,10 @@ public override string ToString()
             builder.Append("}");
             return builder.ToString();
         }
+
+        private bool DefaultIsOriginAllowed(string origin)
+        {
+            return Origins.Contains(origin);
```

Comment:
@blowdart. Happy to add this, but note I lifted this straight out of the `CorsService`. This was the logic that was previously being used to evaluate where an origin is allowed, but obviously this has now been replaced with a call to the `IsOriginAllowed` func on the policy and I am just setting this as the default behaviour. So this would potentially have a subtle change on the default behaviour for checking origins.